packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.styles.ts

```diff
@@ -349,19 +343,24 @@ export const getStyles = memoizeFunction(
         outline: 'none',
         font: 'inherit',
         textOverflow: 'ellipsis',
-        padding: '0'
+        padding: '0',
+        selectors: {
+          '::-ms-clear': {
+            display: 'none'
+          },
+          '::placeholder': placeholderStyles, // Chrome, Safari, Opera, Firefox
+          ':-ms-input-placeholder': placeholderStyles, // IE 10+
+          '::-ms-input-placeholder': placeholderStyles // Edge

```

Comment:
Will these 3 always be together for any styled input placeholder? If so I think it might be worthwhile to put this into a helper/mixin somewhere since anything with a text input would want similar styling.

Reply:
@joschect I was thinking of having a function like `getPlaceholderStyles()` that takes in an `IStyle` as input and returns a `[key: string]: IStyle` for each of the browsers, and exporting it from `[key: string]: IStyle`, so it usable by all of the components. What are your thoughts on this?

Reply:
Made a commit for this https://github.com/OfficeDev/office-ui-fabric-react/pull/9488/commits/dc90d7e1a2092cc4ee787a00301eaae2ab1f3622